apps/jsonrpc2/lib/jsonrpc2/servers/tcp/protocol.ex

```diff
@@ -0,0 +1,59 @@
+defmodule JSONRPC2.Servers.TCP.Protocol do
```

Comment:
What is the non-websockets non-HTTP TPC protocol used for?

Reply:
unix domain sockets

Reply:
unix domain sockets